libs/offline-tasks/src/main/java/org/opensearch/offline_tasks/task/TaskStatus.java

```diff
@@ -0,0 +1,34 @@
+/*
+ * SPDX-License-Identifier: Apache-2.0
+ *
+ * The OpenSearch Contributors require contributions made to
+ * this file be licensed under the Apache-2.0 license or a
+ * compatible open source license.
+ */
+
+package org.opensearch.offline_tasks.task;
+
+import org.opensearch.common.annotation.ExperimentalApi;
+
+/**
+ * Task status enum
+ */
+@ExperimentalApi
+public enum TaskStatus {
+
+    /**
+     * TaskStatus of an in progress Task
+     */
+    ACTIVE,
```

Comment:
Do we also need to define status for ready to be picked tasks?

Reply:
yes, thanks for point this out. Let me add that